pkg/vadmin/replication_start_vc_test.go

```diff
@@ -27,43 +27,43 @@ import (
 )
 
 // mock version of VReplicateDatabase() that is invoked inside VClusterOps.ReplicateDB()
-func (m *MockVClusterOps) VReplicateDatabase(options *vops.VReplicationDatabaseOptions) (err error) {
+func (m *MockVClusterOps) VReplicateDatabase(options *vops.VReplicationDatabaseOptions) (a int64, err error) {
```

Comment:
The replication changes in 09/16 vclusterOps will fail vrep e2e test. Use main branch of vcluster in go.mod.